robot/description/launch/caffeine_gazebo.launch

```diff
@@ -54,6 +54,6 @@
 
    <!-- Run Twist Multiplexer -->
    <include file="$(find twist_mux)/launch/twist_mux.launch" ns="caffeine">
-      <rosparam name="cmd_vel_out" value="cmd_vel"/>
+      <arg name="cmd_vel_out" value="cmd_vel"/>
```

Comment:
Good catch! `<arg>` is passed to the called launch file, whereas `<rosparam>` is passed to the constructor/main function of the called node